module/stats.js

```diff
@@ -165,15 +166,13 @@ const toggleData = async ( element ) => {
 	// Toggle the side of the switch icon
 	element.classList.toggle( 'fa-rotate-180' );
 
-	// Load data from the API / local storage
-	let data = await module.fetchData()
-
-	// If data is not available, stop execution
-	if ( !data ) return
+	// Load data from the local storage because its a toggle
+	let data = await app.getLocalStorageByKey(storageKey);
+	// If data is not available, then fetch from api
+	if ( !data )
+		data = await module.fetchData();
```

Comment:
I was thinking there can be cases where result can be change when we toggle between, for example global results can be change drastically. That's why decided to fetch from API all the time. What do you think? 🙂

Reply:
Yes. But, the health bureau only updates it once or twice a day. But, we can fetch every toggle. Shall I revert it?

Reply:
If there are any new cases, they update it right? Let's keep this for now? What do you think? 🙂

Reply:
Ok. Keeping this for now.

Reply:
My bad communication, let's rollback to the previous version all fetch data from api always 🙂

Reply:
Done.